docs/content/components/autocomplete.md

```diff
@@ -56,30 +222,75 @@ The `.suggester` component is meant for showing suggestions while typing in a la
 
 ```html live
 <div class="form-group position-relative">
-  <textarea class="form-control width-full" placeholder="Leave a comment" aria-label="Comment body">This is on top of #</textarea>
+  <textarea class="form-control width-full" placeholder="Leave a comment" aria-label="Comment body">
+This is on top of #</textarea
+  >
   <ul class="suggester suggester-container" role="listbox" style="top: 4px; left: 125px;">
     <li aria-selected="true">
-      <svg class="octicon color-fg-subtle" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path d="M8 9.5a1.5 1.5 0 100-3 1.5 1.5 0 000 3z"></path><path fill-rule="evenodd" d="M8 0a8 8 0 100 16A8 8 0 008 0zM1.5 8a6.5 6.5 0 1113 0 6.5 6.5 0 01-13 0z"></path></svg>
+      <svg
```

Comment:
A lot of this was just my local Prettier restyling the code - no actual code changes!